File: javascript/7kyu - D.js
//Link -> https://www.codewars.com/kata/53670c5867e9f2222f000225

function isOrthogonal(u,v) {
    const vectorDots = u.length - 1;
    var dotProduct = 0;

    for(i = 0; i <= vectorDots; i++){
        dotProduct += u[i] * v[i];
    }

    if(dotProduct == 0){
        return true;
    } else {
        return false;
    }
}

File: python/7kyu - D.py
# Link -> https://www.codewars.com/kata/55d17ddd6d7868493e000074/python

class Node(object):
    def __init__(self, data):
        self.data = data
        self.next = None

def append(listA, listB):
    currentNode = listA
    
    if currentNode is None:
        return listB
    elif listB is None:
        return listA
    
    while currentNode.next is not None:
        currentNode = currentNode.next
    
    currentNode.next = listB
    
    return listA
